auth/README.md

```diff
@@ -74,6 +74,10 @@ default values.
 | MF_AUTH_SECRET                | String used for signing tokens                                           | auth           |
 | MF_AUTH_LOGIN_TOKEN_DURATION  | The login token expiration period                                        | 10h            |
 | MF_JAEGER_URL                 | Jaeger server URL                                                        | localhost:6831 |
+| MF_KETO_READ_REMOTE_HOST      | Keto Read Host                                                           | mainflux-keto  |
```

Comment:
Is it OK that they have identical host/container names? These are two different containers if I understand well.

Reply:
Same container.
In docker deployment keto expose read and write on the same host with different ports.

Reply:
This PR doesn't change anything in keto deployment. It just enables Auth service to have separate URLs for read and write connection to Keto. That wasn't  possible because Auth used the same environment variable for both read and write connection

Reply:
Can it be 2 different hosts? If not, then use one host with 2 ports, i.e. add just one variable: `MF_KETO_REMOTE_HOST` and two env vars for ports: `MF_KETO_READ_REMOTE_PORT` and `MF_KETO_WRITE_REMOTE_PORT` (additionally - do we need word "REMOTE" in these and do we use it in other var names?).

Reply:
@drasko  What you propose is how it is now.
But we need Auth service to be capable to connect to two different keto hosts, not just the same host and different ports.
That is because in k8s setup keto read and write endpoints are on different hosts.
So, we can have like in this PR four envars (for read host, read port, write host, write port) or maybe two envars (each in `host:port` format). In any way we need possibility to configure differently both host and port part of connection urls

Reply:
OK, thanks